src/components/Task.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 interface Task {
   title: string;
   completed: boolean;
-  key: string;
+  id: string;
```

Comment:
Avoid naming confusion with React specific language if possible.
`itemId` is also probably fine, but since the only we care about in this app is Tasks, we can use the common `id`.